### PR TITLE
1.0.1

### PR DESCRIPTION
**Pull request by mitchellmanware**

### CODE REVIEW

**Comment by mitchellmanware** [comment]
@kyle-messier
It seems like the checks are not consistently handling the license stub note. Some have not problem with the "License: MIT + file LICENSE" line (https://github.com/NIEHS/amadeus/actions/runs/10153735900/job/28077640433) whereas others throw notes for this line or "License: MIT" only (https://github.com/NIEHS/amadeus/actions/runs/10153752290/job/28077688790). ChatGPT said what we have now should be fine so I am not entirely sure how to fix this one.

I also updated the vignette to not run live code and return pre-written messages to mimic output, as the reading of .nc files for the workflow vignette was causing the checks to run indefinitely. Not a perfect solution but hopefully good enough for now, and keeps the package size reasonable (4.7 Mb).

I will resubmit this version to CRAN shortly, but I will wait to pull into main until accepted.

**Comment by kyle-messier** [comment]
@mitchellmanware How do the LICENSE updates do with the `winbuilder` check? If the `--as-cran` checks pass, then we should be all good. I'm also fine with submitting and seeing what gets kicked back and dealing with it that way. Thanks!

File: DESCRIPTION
Comment (kyle-messier) [inline DESCRIPTION:1]: Is adding `tigris` as a depends necessary for the vignette to pass? 

Also, `MIT + license file` line is still the same. I thought we were going to remove the `+ license file` section of that?

File: vignettes/workflow.Rmd
Comment (kyle-messier) [inline vignettes/workflow.Rmd:1]: We can look at other examples on CRAN to see how they handled the vignettes to get them to be lightweight. If copying output and putting in an `eval=FALSE` tag is standard, then we are all good.

File: R/calculate_covariates.R
Comment (kyle-messier) [inline R/calculate_covariates.R:1]: Can you run  a  `tictoc` timing test on this example to ensure it abides by the suggested time?

File: DESCRIPTION
Comment (mitchellmanware) [reply DESCRIPTION:1]: Yes. Packages used in vignettes need to be added to `Suggests` if not already included in the package dependencies.

I tried using `License: MIT`, but it still returned a note.
Screenshot 2024-07-30 at 7 57 52 AM
ChatGPT says `License: MIT + file LICENSE` should be fine so I will add a comment to the CRAN submission to address this note.

File: R/calculate_covariates.R
Comment (mitchellmanware) [reply R/calculate_covariates.R:1]: `tictoc` results below. I changed the source point sample size from 100 to 10, and the radius from 1e5 to 1e3.

```
> tic("calc_sedc")
> library(terra)
> library(sf)
> set.seed(101)
> ncpath <- system.file("gpkg/nc.gpkg", package = "sf")
> nc <- terra::vect(ncpath)
> nc <- terra::project(nc, "EPSG:5070")
> pnt_locs <- terra::centroids(nc, inside = TRUE)
> pnt_locs <- pnt_locs[, "NAME"]
> pnt_from <- terra::spatSample(nc, 10L)
> pnt_from$pid <- seq(1, 10)
> pnt_from <- pnt_from[, "pid"]
> pnt_from$val1 <- rgamma(10L, 1, 0.05)
> pnt_from$val2 <- rgamma(10L, 2, 1)
> 
> vals <- c("val1", "val2")
> calc_sedc(pnt_locs, pnt_from, "NAME", 1e5, vals)
Joining with `by = join_by(from_id)`
Joining with `by = join_by(to_id)`
Joining with `by = join_by(from_id, to_id)`
            NAME      val1      val2
1       Alamance 117.44859 13.971186
...
100       Yancey  16.65552  2.577881
> toc()
calc_sedc: 0.139 sec elapsed
```

File: DESCRIPTION
Comment (mitchellmanware) [reply DESCRIPTION:1]: I also tried to edit the LICENSE file itself, thinking the brackets in `{SET}group` could be causing the NOTE, but it persists.
Screenshot 2024-07-30 at 9 30 58 AM

**Comment by kyle-messier** [comment]
@mitchellmanware I'd say let's resubmit to CRAN and see what happens. Perhaps there will be 1 or 2 unresolved notes. Regardless, it would be good to reset the resubmission clock.

**Review by kyle-messier — APPROVED** [review]

**Pull request merged**